modules/nf-core/spaceranger/count/tests/main.nf.test

```diff
@@ -68,7 +68,7 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    process.out.versions,
+                    path(process.out.versions.get(0)).yaml,
```

Comment:
This is a good addition to the traditional way of checking the versions file.
Unfortunately it does crash the linting.
You just need to add a [`versions` key in the snapshot](https://github.com/nf-core/tools/blob/41b4516e1b5dedca816749ca2a0019254cab4831/nf_core/modules/lint/module_tests.py#L140C32-L140C116) to fix it.

Reply:
Would something like `[versions:path(process.out.versions[0]).yaml]` works ?

Reply:
> lints should be ignored(ie the "missing" versions, 

It's checking for a "versions" snapshot. https://github.com/nf-core/modules/blob/8e4537e40554ae77da23d4ed290be1362fa8a611/modules/nf-core/antismash/antismashlite/tests/main.nf.test.snap#L2-L13. Which I actually thought was an issue was you could only have one, but you actually can? https://github.com/askimed/nf-test/issues/197

Yeah, I have an issue out for it #6505, and then a PR for a hook to update those #7098 .

I don't know. Maybe I'm bad at Python, but adding new linting rules within tools is challenging. So some of these things lag because there's a better way of doing something. (In this case, the hashes were failing for me, but does the .yaml calls worked?)

Kinda a meta-issue. It's a good guardrail, but I'm arguing we should ignore it here.